docs/guides/postinstall.md

```diff
@@ -187,3 +188,63 @@ S3 suspend has been broken since macOS Sonoma, it has never been fixed, but this
 !!! note
     Make sure you have `CONFIG_MODULE_FORCE_UNLOAD=y` in the kernel config.
     To check, run: `zcat /proc/config.gz | grep "CONFIG_MODULE_FORCE_UNLOAD"` on Arch based distros.
+
+### Gentoo/OpenRC
+
+S3 suspend has been broken since macOS Sonoma, it has never been fixed, but this workaround will make deep suspend work on Gentoo Linux using OpenRC and elogind.
+
+Prerequisites:
+1. Make sure elogind is installed and running:
+    ```bash
```

Comment:
there should be a blank line before and after a code block as per lint rules

Reply:
5 spaces should be for indentation of a code block in a point, you have added 4. PLEASE TAKE THE OTHER CODE BLOCKS IN THIS GUIDE AS THE REFERENCE.